roottest/root/io/dirdelete/CMakeLists.txt

```
@@ -1,7 +1,3 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-ROOTTEST_ADD_OLDTEST(LABELS longtest)
+ROOTTEST_ADD_TEST(dirdelete
+                  MACRO rundirdelete.C
+                  OUTREF dirdelete.fail.ref)
```

Here we lose the information that the test is actually failing.  We could either leave it as is or we could use:

```suggestion
                  OUTREF dirdelete.ref
                  WILLFAIL)
```
either way the issue is now reported at https://github.com/root-project/root/issues/20314

---

Moving discussion to the issue